util/chunk/list.go

```diff
@@ -83,11 +92,10 @@ func (l *List) AppendRow(row Row) RowPtr {
 func (l *List) Add(chk *Chunk) {
 	// FixMe: we should avoid add a Chunk that chk.NumRows() > list.maxChunkSize.
 	if chk.NumRows() == 0 {
-		panic(" add empty chunk")
-	}
-	if chkIdx := len(l.chunks) - 1; chkIdx != -1 {
-		l.memUsageChunks += l.chunks[chkIdx].MemoryUsage()
+		panic("chunk appended to List should have at least 1 rows")
 	}
+	l.memTracker.Consume(chk.MemoryUsage())
```

Comment:
If Add and AppendRow is used together, the last chk of List seems not be consumed.